src/eql.erl

```diff
@@ -63,3 +73,24 @@ get_query(Name, Proplist) ->
         {Name, Value} -> {ok, Value};
         false -> undefined
     end.
+
+%% internal functions
+
+-spec file_to_namespace(file:filename_all()) -> atom().
+file_to_namespace(File) ->
+    list_to_atom(filename:basename(File, ".sql")).
```

Comment:
maybe `rootname(basename(Filename))`? because we support «not only sql» :)
or use full file name with original type, like `eql:get_query({"queries.sql", get_all_users}, test_tab)`, what do you think?

Reply:
Hm, ok, I'd say `rootname` to be able to keep it as an unquoted atom. I'll change it to that tomorrow.